{
  "comments": [
    {
      "key": {
        "uuid": "da33351e_5179602e",
        "filename": "utils/test/testapi/tools/watchdog/docker_watch.sh",
        "patchSetId": 2
      },
      "lineNbr": 76,
      "author": {
        "id": 335
      },
      "writtenOn": "2017-08-16T08:40:22Z",
      "side": 1,
      "message": "I just restart the test api this morning\ndocker was fine and reporting conatiner was fine, only testapi was down",
      "revId": "6eba5d00018987a2021e886c4b8c2ba9ddf86a8b",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "da33351e_c974a09e",
        "filename": "utils/test/testapi/tools/watchdog/docker_watch.sh",
        "patchSetId": 2
      },
      "lineNbr": 76,
      "author": {
        "id": 6433
      },
      "writtenOn": "2017-08-22T07:33:38Z",
      "side": 1,
      "message": "Yes, but sometimes docker hangs on ports even after running docker stop command. So, for that we need to restart the docker.",
      "parentUuid": "da33351e_5179602e",
      "revId": "6eba5d00018987a2021e886c4b8c2ba9ddf86a8b",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "da33351e_9a776fd4",
        "filename": "utils/test/testapi/tools/watchdog/docker_watch.sh",
        "patchSetId": 2
      },
      "lineNbr": 76,
      "author": {
        "id": 2651
      },
      "writtenOn": "2017-08-22T07:59:34Z",
      "side": 1,
      "message": "in this case you can check if docker-proxy which occupy 8082 is still running, if so, kill the process first, then resume testapi container",
      "parentUuid": "da33351e_c974a09e",
      "revId": "6eba5d00018987a2021e886c4b8c2ba9ddf86a8b",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "da33351e_65b12ce1",
        "filename": "utils/test/testapi/tools/watchdog/docker_watch.sh",
        "patchSetId": 2
      },
      "lineNbr": 76,
      "author": {
        "id": 6433
      },
      "writtenOn": "2017-08-26T08:42:30Z",
      "side": 1,
      "message": "Done",
      "parentUuid": "da33351e_9a776fd4",
      "revId": "6eba5d00018987a2021e886c4b8c2ba9ddf86a8b",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "da33351e_6625c8c1",
        "filename": "utils/test/testapi/tools/watchdog/docker_watch.sh",
        "patchSetId": 2
      },
      "lineNbr": 80,
      "author": {
        "id": 2651
      },
      "writtenOn": "2017-08-16T06:12:45Z",
      "side": 1,
      "message": "1) remove the container and docker run a new one wouldn\u0027t be recommended, it will erase logs/exceptions/tracks. restart docker daemon and containers directly would be recommended.\n\n2) there are two levels of crash so far, only testapi/reporting(potentially) stops working, or docker daemon works abnormally. Wouldn\u0027t it be better to have two level\u0027s recovery, restart stopped container only, or restart docker daemon and should-be-running containers?",
      "revId": "6eba5d00018987a2021e886c4b8c2ba9ddf86a8b",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "da33351e_1a96bff0",
        "filename": "utils/test/testapi/tools/watchdog/docker_watch.sh",
        "patchSetId": 2
      },
      "lineNbr": 80,
      "author": {
        "id": 6433
      },
      "writtenOn": "2017-08-22T07:33:38Z",
      "side": 1,
      "message": "Done",
      "parentUuid": "da33351e_6625c8c1",
      "revId": "6eba5d00018987a2021e886c4b8c2ba9ddf86a8b",
      "serverId": "bbac25d2-bf60-4904-9ba8-a72fc000d6c5",
      "unresolved": false
    }
  ]
}